docs/_components/dropzone.md

```diff
@@ -0,0 +1,638 @@
+---
+layout: page
+title: DropZone
+category: Components
+---
+
+The DropZone is a component that can create a pre-formatted area for users to drag files, it can also be used to convert an existing HTML element into a droppable area. 
+
+## Example usage
+
+### Vanilla
+
+DropZone using default configuration.
+
+![DropZoneComponent vanilla drop]({{ site.baseurl }}/assets/image_examples/dropzone-vanilla-drop.gif){: .dropzone-example }
+
+{% raw %}
+```twig
+{% import '@pulsar/pulsar/v2/helpers/html.html.twig' as html %}
+
+{{
+    html.dropzone({
+        'id': 'drop-zone'
+    })
+}}
+```
+```javascript
+import DropZoneComponentFactory from '../path/to/DropZoneComponentFactory';
+
+const dropZoneComponent = DropZoneComponentFactory.create(
+    // <html> element
+    document.documentElement, 
+    // selector to identify DropZone elements
+    '.dropzone'
+);
+
+dropZoneComponent.init();
+```
+{% endraw %}
+
+---
+
+### Passive
+
+DropZone initiated in passive mode.
```

Comment:
I've moved the passive description up to here, makes much more sense in this section.